apps/data/lib/data/opportunities.ex

```diff
@@ -4,15 +4,16 @@ defmodule Data.Opportunities do
   """
 
   alias Data.{Opportunity, Repo}
-  require Ecto.Query
+  import Ecto.Query, except: [update: 2]
 
-  @defaults %{include: [], sort_by: :inserted_at, page: 1, page_size: 25}
+  @defaults %{filters: %{}, include: [], sort_by: :inserted_at, page: 1, page_size: 5}
```

Comment:
Reset `:page_size`

Reply:
is 5 too low? I feel like 10 is a good sane default